client/wc-api/reports/items/selectors.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { isNil } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getResourceName } from '../../utils';
+import { DEFAULT_REQUIREMENT } from '../../constants';
+
+const getReportItems = ( getResource, requireResource ) => (
+	type,
+	query = {},
+	requirement = DEFAULT_REQUIREMENT
+) => {
+	const resourceName = getResourceName( `report-items-query-${ type }`, query );
+	const data = requireResource( requirement, resourceName ) || {};
+
+	return data;
```

Comment:
Small suggestion: we can get rid of the `data` assignment and return its value directly.
```ES6
	return requireResource( requirement, resourceName ) || {};
```

Reply:
Done in 7f934aa.